### PR TITLE
Use snapshots to test fields rendering of contrib mixins

### PR DESCRIPTION
**Pull request by jeromelebleu**
To simplify tests and ease the debugging and the maintenance, this adds a generic mixin `FormFieldsSnapshotTestMixin` to validate the rendering of form fields using snapshots. Tests of Bootstrap and Foundation are updated to make use of it.

It is using [pytest-snapshot](https://github.com/joseph-roitman/pytest-snapshot) to verify and generate snapshots.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -1,27 +1,21 @@
 [[source]]
-
 url = "https://pypi.python.org/simple"
 verify_ssl = true
 name = "pypi"
 
-
 [dev-packages]
-
 pytest = "*"
```
Comment (stephrdev) [inline Pipfile:7]: Could we use the PR to also push the minimal version the other dev dependencies to something newer?

**Comment by jeromelebleu** [comment]
> Sorry for getting back to this that late.

No problem, I guess this particular time doesn't help and hope you are fine.

> Would love to merge if we take the chance to bump the other dev dependencies too.

I don't rely know pipenv, is `pipenv lock` - as done in 3fb3810 - enough?

Also, to pass the tests it could be useful to merge #13 at first.

**Comment by stephrdev** [comment]
> I don't rely know pipenv, is `pipenv lock` - as done in [3fb3810](https://github.com/stephrdev/django-tapeforms/commit/3fb3810fc6a5bb59325213b3dda2ebdf586bbbcd) - enough?
You can simply add the version specifiers ">=x.x" to the Pipfile entries and afterwards lock.

> Also, to pass the tests it could be useful to merge #13 at first.
Yep, we're into that already :)

**Comment by jeromelebleu** [comment]
I have updated Python and Django versions, tell me if more things should be done.

**Comment by stephrdev** [comment]
I think bumping the dev dependencies could be good too. pytest and so on. Besides that we're ready to merge.

**Comment by jeromelebleu** [comment]
> I think bumping the dev dependencies could be good too. pytest and so on.

Done with d91a009, I also rebased on master.

**Pull request merged**